test.js

```diff
@@ -4,11 +4,10 @@ const os = require('os')
 const path = require('path')
 const klaw = require('klaw')
 const Mocha = require('mocha')
-const assign = require('./lib/util/assign')
 
 const argv = require('minimist')(process.argv.slice(2))
 
-const mochaOpts = assign({
+const mochaOpts = Object.assign({
```

Comment:
How about:

```js
const mochaOpts = {
  ...argv,
  ui: 'bdd',
  reporter: 'dot',
  timeout: 30000
};

Reply:
@JPeer264 That's object spread properties; which are only supported in Node 8+, so we can't use those yet. Someday, though...

Reply:
Oh dang! I guess I am using too much babel in my projects. Good to know though ;)